### PR TITLE
Added backward option for Bidirectional RNN

### PR DESCRIPTION
**Pull request by dbonadiman**
I reposted the pull request fixing the issues and removing some clutter

### CODE REVIEW

File: examples/imdb_bidirectional_lstm.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+model.add_input(name='input', input_shape=(1,), dtype=int)
+model.add_node(Embedding(max_features, 128, input_length=maxlen), name='embedding', input='input')
+model.add_node(LSTM(64), name='forward', input='embedding' )  # You can change these two layers with GRU
+model.add_node(LSTM(64, go_backwards=True), name='backward', input='embedding' )
+model.add_node(Dropout(0.5), name='dropout', inputs=['forward', 'backward'])
+model.add_node(Dense(1, activation='sigmoid'), name='sigmoid', input='dropout')
+model.add_output(name='output', input='sigmoid')
+
+# try using different optimizers and different optimizer configs
+model.compile('adam',{'output':'binary_crossentropy'})
```
Comment (fchollet) [inline examples/imdb_bidirectional_lstm.py]: Space (PEP8).

**Comment by fchollet** [comment]
One important thing to fix: the new argument should be added the layers' config in `get_config`.

**Comment by dbonadiman** [comment]
Done, now should be ok.

**Comment by fchollet** [comment]
Merged. Now can you update the documentation for the recurrent layers to include the `go_backwards` argument (in a separate PR)? Thanks : )

**Comment by dbonadiman** [comment]
sure, probably tonight (Italy time) out of works hours.

**Pull request merged**